src/rattler_build_conda_compat/jinja/utils.py

```diff
@@ -0,0 +1,11 @@
+from jinja2 import DebugUndefined
+
+
+class _MissingUndefined(DebugUndefined):
```

Comment:
Cant this error be returned by the functions to the user? In that case I think this thing shouldnt be private.

Reply:
I think you should in that case also either not put it in the `utils` module or reexport it.

Reply:
This error is not returned to the user ( jinja is raising directly jinja2.Undefined ) and it's something specific for our error handling - that's why I would prefer to keep private